bigchaindb_driver/driver.py

```diff
@@ -353,15 +354,16 @@ class OutputsEndpoint(NamespacedDriver):
     """
     PATH = '/outputs/'
 
-    def get(self, public_key, unspent=False, headers=None):
+    def get(self, public_key, spent=None, headers=None):
         """
 
         Args:
             public_key (str): Public key for which unfulfilled
                 conditions are sought.
-            unspent (bool): Whether to get the unconsumed outputs
-                only. Defaults to ``False``, meaning that both consumed,
-                and unconsumed outputs will be returned.
+            spent (bool): Indicate if the result set should include only spent
+                or only unspent outputs. If not specified (``None``) the
+                result includes all the outputs (both spent and unspent)
+                associated with the public_key
```

Comment:
Very small details: 

1. public key or ``public_key``
2. end of sentence with a period "."

Sorry for being so picky :)